enterprise/models.py

```diff
@@ -1290,20 +1290,20 @@ class EnterpriseCustomerIdentityProvider(TimeStampedModel):
     .. no_pii:
     """
 
-    enterprise_customer = models.OneToOneField(
+    enterprise_customer = models.ForeignKey(
```

Comment:
This has been changed to foreign key so that one enterprise customer can be linked to multiple identity providers.